src/web/mjs/connectors/ManHuaGui.mjs

```diff
@@ -14,6 +14,37 @@ export default class ManHuaGui extends SinMH {
         this.pathMatch = '/list/index_p(\\d+).html';
         this.queryMangasPageCount = 'div.pager-cont div.pager a:last-of-type';
         this.queryChapters = 'div.chapter-list ul li a';
+        this.config = {
+            throttle: {
+                label: 'Chapter Throttle Requests [ms]',
+                description: 'Enter the timespan in [ms] to delay consecuitive HTTP requests while downloading Chapters.\nThe website may ban your IP for to many consecuitive requests.',
+                input: 'numeric',
+                min: 500,
+                max: 10000,
+                value: 2500
+            },
+            delaylist: {
```

Comment:
Why not hardcoding e.g. 5000 instead of making a user setting

Reply:
That's also a good idea.. But, if the site changes the required throttle then users have to create a new issue again...